cli/src/main/package/bin/ide.bat

```diff
@@ -17,6 +33,7 @@ REM https://stackoverflow.com/questions/61888625/what-is-f-in-the-for-loop-comma
 for /f "tokens=*" %%i in ('ideasy %IDE_OPTIONS% env') do (
   call set %%i
 )
-if not %ERRORLEVEL% == 0 (
+
+if %ERRORLEVEL% == 0 (
   echo IDE environment variables have been set for %IDE_HOME% in workspace %WORKSPACE%
```

Comment:
> > https://github.com/devonfw/IDEasy/blob/d36317e3436615b18d0be121ee96e5201b896677/cli/src/main/package/bin/ide.bat#L27-L29
> > 
> > Is the condition right here?
> 
> Very good point. We actually want the exit code of `ideasy env` command but we are directly processing the output of it and invoking `call set %%i` in the loop. Maybe the `ERRORLEVEL` will be overridden by that command inside the loop. Coding `CMD` is actually even 10 dimensions worse to coding `bash` what can already be some kind of nightmare itself. But solving this problem in `bash` would be very easy and doing the same in `CMD` could be tricky - at least I have no clue how to do it. I just did a simple test:
> 
> ```
> C:\Users\hohwille>ide
> 
> 
> C:\Users\hohwille>d:
> 
> D:\>cd projects
> 
> D:\projects>cd IDEasy
> 
> D:\projects\IDEasy>ide
> 
> D:\projects\IDEasy>
> ```
> 
> So it seems this condition is never met and the message is never printed out at all. We need to create a separate bug issue for this.
> 
> UPDATE: I created bug #894 for this problem since it is unrelated to the GIT PATH issue.

I think the removing of `not` is the only fix we need. I get the `ERRORLEVEL` of 17, when I run the `ide env` outside a project folder and there is not output. The only thing remains, is that if we use `ide` in a project folder, we get the `echo IDE environment variables have been set for %IDE_HOME% in workspace %WORKSPACE%` message as output